### DISCUSSION TITLE
Item Brainstorm

### DISCUSSION THREAD

**Discussion by ntseng** [Imaginary-Horizons-Productions/BountyBot#198] [General]
- [x] Bounty Reward Boost
- [ ] Custom Bounty Image (remove default functionality)
- [x] Custom Bounty Thumbnail
- [ ] Custom Bounty Event (remove default functionality)
- [x] Per Server XP Gain
- [x] Achievement Progress Item (Lootbox?)
- [x] Summon Raid Boss
- [ ] Brag Button

**Comment by ntseng** [comment]
- [ ] XP Bonus in this server for X days

**Comment by SillySalamndr** [comment]
Ideas for toasts?
 - [ ] Super Toasts?
 - [ ] "And you too!"?

**Comment by ntseng** [comment]
Ascension Orb
Resets the user's level in the used server to 1, then adds a permanent XP Bonus for the user in that server.
